tests/PHPStan/Rules/Functions/data/variadic-parameters-declaration.php

```diff
@@ -0,0 +1,19 @@
+<?php declare(strict_types=1);
+
+namespace VariadicParametersDeclaration;
+
+class HelloWorld
+{
+	public function multipleVariadicParams(int ...$p1, int ...$p2): void
```

Comment:
It's always a good idea to include also an example where the rule is not supposed to report anything.